doc/versions.json

```diff
@@ -16,6 +17,12 @@
       "zephyr": "2.3.0-rc1",
       "mcuboot": "1.6.0-rc2"
     },
+	"1.3.1": {
+      "ncs": "1.3.1",
+      "nrfxlib": "1.3.1",
+      "zephyr": "2.3.0-rc1",
+      "mcuboot": "1.6.0-rc2"
```

Comment:
Is it the correct mcuboot version though?

Reply:
That was the mcuboot version for 1.3.0, and mcuboot wasn't updated for 1.3.1 - correct @shantha-14 ?

Reply:
Shantha confirms there is no change from 1.3.0.